src/main/java/com/pnu/dev/shpaltaif/repository/specification/SearchCriteria.java

```diff
@@ -0,0 +1,15 @@
+package com.pnu.dev.shpaltaif.repository.specification;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
```

Comment:
Do we really need `@NoArgsConstructor` and `@AllArgsConstructor` here?

Reply:
Sorry, only `@AllArgsContructor`